Source/WebKit/Platform/spi/Cocoa/AuthenticationServicesCoreSPI.h

```diff
@@ -30,7 +30,9 @@
 @interface ASCWebKitSPISupport : NSObject
 @property (class, nonatomic) BOOL shouldUseAlternateCredentialStore;
 + (BOOL)arePasskeysDisallowedForRelyingParty:(nonnull NSString *)relyingParty;
++ (void)getArePasskeysDisallowedForRelyingParty:(nonnull NSString *)relyingParty withCompletionHandler:(void (^ _Nonnull)(BOOL))completionHandler NS_REFINED_FOR_SWIFT;
 + (BOOL)canCurrentProcessAccessPasskeysForRelyingParty:(nonnull NSString *)relyingParty;
```

Comment:
Why do you leave a declaration for `canCurrentProcessAccessPasskeysForRelyingParty` in the header, when you seem to remove it in the implementation file?

Reply:
I thought getArePasskeysDisallowedForRelyingParty and it's sync equivalent were introduced at different times, but it turns out that's not true. I've removed usage of canCurrentProcessAccessPasskeysForRelyingParty now.